src/test.ts

```diff
@@ -34,10 +34,13 @@ export async function test(fn_name: string, test_req: any): Promise<string> {
       headers,
     });
     const response = await request;
-    console.log('request:', request);
     if (typeof request.text !== 'function') {
```

Comment:
I know this wasn't your PR, but any ideas why this is here?

If it isn't clear, I mean the check for `typeof request.text !== 'function'`